src/System.ServiceProcess.ServiceController/tests/SafeServiceControllerTests.cs

```diff
@@ -167,6 +167,7 @@ public static void SetServiceName_GetDisplayName()
         }
 
         [Fact]
+        [OuterLoop]
```

Comment:
This should be marked as "[ActiveIssue(39216)]" instead of being marked as "OuterLoop" in order to disable this flaky test. Otherwise, OuterLoop testing will become flaky.

And you can then leave #39216 active and mark it as "Disabled Test" label.

cc: @danmosemsft @stephentoub

Reply:
No we shouldn't mark it with active issue as we are going to close the issue because the underlying problem is coming from Windows and we have reported the issue to Windows. We are marking this as outerloop to avoid the problem in regular runs.

Reply:
CC @jkotas

Reply:
>We are marking this as outerloop to avoid the problem in regular runs.

But then we have 'OuterLoop' being flaky.  We should disable this test forever in that case.

Reply:
Not really as the problem happen only when having more than UI language and you didn't boot the machine after enabling the UI language. it is very unlikely scenario but it can happen on individuals machines as it did on @jkotas case.